### PR TITLE
Added excludes and made fixes from coverage testing for 5.0 release.

### PR DESCRIPTION
**Pull request by henryso**
See corresponding tests in gregorio-project/gregorio-test#284.

Please review and merge if satisfactory.

### CODE REVIEW

**Comment by eroux** [comment]
Thanks a lot! Looks good to me, although I admit there are a few things I didn't understand, but it's not blocking, it's ok for me to merge

File: src/gabc/gabc-write.c [hunk 1]
```diff
@@ -231,7 +231,7 @@ static void gabc_print_char(FILE *f, const grewchar to_print)
 {
     if (write_state == GABC_AT_PROTRUSION_FACTOR) {
         write_state = GABC_IN_PROTRUSION_FACTOR;
-        if (to_print == ':') {
+        if (to_print == 'd') {
```
Comment (eroux) [inline src/gabc/gabc-write.c]: this change is the most mysterious to me...

Comment (henryso) [inline src/gabc/gabc-write.c]: This is a bug in the gabc->gabc translation that was made apparent by the coverage testing.  The parser uses `d` for the argument in the internal structure when `<pr>` (no argument) is used.

Comment (eroux) [inline src/gabc/gabc-write.c]: I admit I don't get it... can you point me to the line in the source that sets something to 'd'? (this sounds like an uninitialized variable?)

Comment (henryso) [inline src/gabc/gabc-write.c]: See https://github.com/gregorio-project/gregorio/blob/develop/src/gabc/gabc-score-determination.y#L957

Comment (eroux) [inline src/gabc/gabc-write.c]: Oh, ok, thanks!

**Pull request merged**